Makefile

```diff
@@ -32,8 +32,8 @@ checkisort:
 
 install-pre-commit-hook:
 	@echo "Installing pre-commit hook to git"
-	@echo "Uninstall the hook with pre-commit uninstall"
-	pre-commit install
+	@echo "Uninstall the hook with poetry run pre-commit uninstall"
+	poetry run pre-commit install
```

Comment:
shoud we add ```poetry run pre-commit``` before the install? running this command install pre-commit dependencies, not only git webhook